app.rb

```diff
@@ -0,0 +1,74 @@
+# rubocop:disable Metrics/CyclomaticComplexity
+require './book'
+require './person'
+require './rental'
+require './json_handler'
+require './refactor'
+
+class App
+  attr_reader :books, :persons, :rentals
+
+  def initialize
+    @persons = []
+    @books = []
+    @rentals = []
+  end
+
+  def menu
+    puts 'Please choose an option by entering a number'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all retals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def options(input)
+    case input
+    when '1' then list_all_books
+    when '2' then list_all_persons
```

Comment:
- [ ] As the previous reviewers mentioned your app produces some errors, kindly fix them so your app is working properly
- When listing the people the app crashes and produces this error:
![Screenshot_20221029_022441](https://user-images.githubusercontent.com/25227022/198755627-bf8d6c96-1f67-468f-94f6-73a82a5c635d.png)

- When trying to create a rental the app crashes and produces this error:
![Screenshot_20221029_022528](https://user-images.githubusercontent.com/25227022/198755652-37f4e19e-8482-41b2-ab71-22111da8e388.png)